.ci/openshift-ci/images/Dockerfile.installer

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 Red Hat Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build the image which wraps the kata-containers installation along with the
+# install script. It is used on a daemonset to deploy kata on OpenShift.
+#
+FROM centos:7
```

Comment:
OOI, why 7 and not 8?

Reply:
In really the OpenShift build pipeline will replace `FROM centos:7` with `FROM openshift/release:golang-1.13` which is an image based on RHEL 7. I could have chose a different one, but this works just fine. I keep `centos:7`  because it is close enough to the actual image and it allows me to do some tests locally.